lib/GuestManager.php

```diff
@@ -228,25 +230,39 @@ public function sendEmailInvitation(Room $room, Participant $participant): void
 		]);
 
 		if ($user instanceof IUser) {
-			$subject = $this->l->t('%s invited you to a conversation.', $user->getDisplayName());
+			$subject = $this->l->t('%s invited you to conversation "%s".', [$user->getDisplayName(), $room->getDisplayName('')]);
```

Comment:
Does that require to be positional parameters or something for translation?

Reply:
if multiple are used, it's recommended yeah
```suggestion
			$subject = $this->l->t('%1$s invited you to conversation "%2$s".', [$user->getDisplayName(), $room->getDisplayName('')]);
```